packages/web/src/components/CheckboxField/_CheckboxField.scss

```diff
@@ -0,0 +1,90 @@
+@use 'theme';
+@use '../../tools/typography';
+
+.CheckboxField {
+    max-width: 100%;
+}
+
+.CheckboxField__label {
+    @include typography.generate(theme.$label-typography);
+
+    margin-bottom: theme.$spacing;
+    color: theme.$label-color-default;
+}
+
+.CheckboxField__label--hidden {
+    position: absolute;
+    overflow: hidden;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    clip: rect(0, 0, 0, 0);
+    white-space: nowrap;
+    border: 0;
+}
+
+.CheckboxField__label--required::after {
+    content: '*';
+    color: theme.$label-required-color;
+}
+
+.CheckboxField__input {
```

Comment:
Please check the construction of the field in Figma:

- There should be 3px space around the input so it makes 3 + 18 + 3 = 24px.
- There should be 8px space between the input (including its surrounding space) and label text.
- There should be additional 8px vertical spacing above and below label.

<img width="1253" alt="Snímek obrazovky 2022-03-03 v 20 23 22" src="https://user-images.githubusercontent.com/5614085/156637566-b4c84d9d-c74a-4e31-a9a1-a708314917f7.png">

Reply:
Fixed.